### PR TITLE
Cleanup names

### LINKED ISSUE #5: Clean up naming conventions.
There are some bad naming conventions throughout. We should improve that.

### PR DESCRIPTION
**Pull request by joshuamorton**
closes #5.

This is moving towards 1.0, I think the API should be stable at this point, really all we need now is some internal refactoring, docs, and some tests, would you agree?

### CODE REVIEW

**Comment by iRapha** [comment]
I partially agree. I think we'll only reach 1.0 once buzzmobile has a handful of simulation tests and we've ironed out the kinks involved in making those. I'm sure once we actually have automated simulation tests we will run into api issues that I think are best ironed out before 1.0.

Speaking of, do you think making some of those is something you want to take a stab at? might be useful for figuring out whether pyros is getting there.

File: README.md [hunk 1]
```diff
@@ -22,14 +22,14 @@ and publishes it. This would test that functionality
...
+from pyrostest import RosTest, launch_node, mock_node
 from std_msgs.msg import Float64
 
 class TestAddOne(RosTest):
     @launch_node('pkg', 'add_one')
     def test_add_one(self):
-        with mock_pub('/input', Flaot64, queue_size=0) as p:
-            with check_topic('/output', Float64) as r:
+        with self.mock_pub('/input', Flaot64, queue_size=0) as p:
+            with self.check_topic('/output', Float64) as r:
```
Comment (iRapha) [inline README.md]: +1

**Comment by joshuamorton** [comment]
Pls no.

Also ignore the failing build, its still flaky (although a lot less flaky than before), I'm working on figuring out the issue.

File: README.md (hunk 1 above)
Comment (joshuamorton) [inline README.md]: The nice thing about this is now mock_pub and check_topic have access to the rosmaster api in case we care about that later.

**Comment by iRapha** [comment]
that flakyness is def before 1.0 too.

Also simulation tests wouldn't be the worst thing tbh. I think you should try. Esp bc we need to figure out a way to have pass/fail conditions. (i.e.: is the car at location x,y at time t or something, or does the car ever crash, etc)

**Comment by joshuamorton** [comment]
I can take a stab, but also I'll need someone to explain how to spin up gazebo.

**Pull request merged**